sched/Kconfig

```diff
@@ -1819,7 +1819,7 @@ config IDLETHREAD_STACKSIZE
 
 config PTHREAD_STACK_MIN
 	int "Minimum pthread stack size"
-	default 256
+	default DEFAULT_TASK_STACKSIZE
```

Comment:
This changes from 256 to 2K - that is not a ok default change to make.

Reply:
256B is normally too small to run a thread, can you any thread on Cortex M4 MCU with so small stack? especially if the interrupt stack is disabled. PTHREAD_STACK_MIN is seldom used macro, we found this issue in one LTP test case which run create a thread with PTHREAD_STACK_MIN(256B) stack and crash immediately.

Reply:
It use to work. 

I just did an uptake and stack penetration has gone way up (>300 bytes) - we need to keep an eye on that!

It is better to up PTHREAD_STACK_MIN to what works not commingle it with DEFAULT_TASK_STACKSIZE.

Reply:
DEFAULT_TASK_STACKSIZE is the default stack size for the normal task, which doesn't have the special(large) stack size requirement, so it's a good candidate for PTHREAD_STACK_MIN. Do you have a better suggestion? At least, 256B is wrong here since the default value can't work with most SoC.

Reply:
BTW, DEFAULT_TASK_STACKSIZE is a reasonable default value for PTHREAD_STACK_MIN because DEFAULT_TASK_STACKSIZE is designed to as a safe minial value for the target platform.